src/react-currency-masked-input.jsx

```diff
@@ -79,3 +79,5 @@ CurrencyMaskedInput.defaultProps = {
   value : null
 }
 
+module.exports = CurrencyMaskedInput;
```

Comment:
True, but compiling this with Babel 6 will result in the `CurrencyMaskedInput` object being an ES6 module of the form:

```javascript
{
  __esModule: true,
  default: CurrencyMaskedInput
}
```

So everywhere people would previously have written, for example, `<CurrencyMaskedInput />`, they would otherwise now have to write `<CurrencyMaskedInput.default />`.  Adding this exports line fixes the issue.  See this blog post for more details: https://medium.com/@kentcdodds/misunderstanding-es6-modules-upgrading-babel-tears-and-a-solution-ad2d5ab93ce0#.yqxsp1qf5